lib/Dialect/TTIR/IR/TTIROps.cpp

```diff
@@ -1012,6 +1012,36 @@ ::mlir::LogicalResult mlir::tt::ttir::AllGatherOp::verify() {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// ReverseOp
+//===----------------------------------------------------------------------===//
+
+::mlir::LogicalResult mlir::tt::ttir::ReverseOp::verify() {
+  auto dimensions = getDimensions();
```

Comment:
We agreed to follow LLVM standards for using auto:
https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable

Can you please switch auto in this case for a real type?